src/ui/login-form-ui-handler.ts

```diff
@@ -233,6 +233,8 @@ export default class LoginFormUiHandler extends FormModalUiHandler {
         globalScene.ui.setOverlayMode(Mode.OPTION_SELECT, {
           options: options,
           delay: 1000,
+          xOffset: globalScene.scaledCanvas.width,
+          yOffset: globalScene.scaledCanvas.height - this.usernameInfoImage.displayHeight - 16 * dataKeys.length - 22,
```

Comment:
NIT: might be worth to destructure `globalScene`

```suggestion
        const { ui, scaledCanvas: { width, height } } = globalScene;
       ui.setOverlayMode(Mode.OPTION_SELECT, {
          options: options,
          delay: 1000,
          xOffset: width,
          yOffset: height - this.usernameInfoImage.displayHeight - 16 * dataKeys.length - 22,
```